docs/batch.rst

```diff
@@ -0,0 +1,122 @@
+################
+Batch operations
+################
+
+.. _batch:
+
+POST /batch
+===========
+
+**Requires an FxA OAuth authentication**
+
+The POST body is a mapping, with the following attributes:
+
+- ``requests``: the list of requests
+- ``defaults``: (*optional*) values in common for all requests
```

Comment:
Reading the docs it's fine, but the name of the argument doesn't convey much more meaning, and I had to read the docs to understand what the purpose was.